scripts/Modules/Saritasa.Cake/Tasks/CleanTask.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Cake.Core;
+using Cake.Frosting;
+
+namespace Saritasa.Cake;
+
+/// <summary>
+/// Clean task.
+/// </summary>
+[TaskName("Clean")]
+[TaskDescription("Clean solution")]
+public sealed class CleanTask : FrostingTask
+{
+    /// <summary>
+    /// Constructor.
+    /// </summary>
+    public CleanTask()
+    {
+        projectForClean = new[]
+        {
+            "src\\Saritasa.Tools.Common",
+            "src\\Saritasa.Tools.Domain",
+            "src\\Saritasa.Tools.EntityFrameworkCore",
+            "src\\Saritasa.Tools.Emails",
+            "src\\Saritasa.Tools.Misc",
+            "test\\Saritasa.Tools.Common.Tests",
+            "test\\Saritasa.Tools.Tests"
+        };
+    }
+
+    /// <summary>
+    /// Project folders to be cleared.
+    /// </summary>
+    private string[] projectForClean;
+
+    /// <inheritdoc />
+    public override void Run(ICakeContext context)
+    {
+        var solutionDir = "..\\..\\..\\";
```

Comment:
Could there be a better way to determine the root directory since it's going to be executed from powershell / bash?